posts/discriminative-vs-generative/index.qmd

```diff
@@ -0,0 +1,595 @@
+---
+title: "Generative vs. discriminative models: what's the difference really?"
```

Comment:
not very convinced by title. how about: "Models: Generative vs. discriminative"?

any better ideas?

Reply:
Agree that the title isn't great. Like your suggestion. Was also thinking something like "Generative vs. Discriminative models -- two approaches to modelling". Not quite convinced about this either but maybe something along those lines 🤷

Reply:
How about just

"Generative vs. Discriminative models"

which together with the subtitle 

"Thinking through the differences between generative and discriminative models."

is more than enough - i think

Reply:
Yup,good for me